views/home.ejs

```diff
@@ -1 +1,32 @@
-<h1> <%= title %> Home Page</h1>
+
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="css\home.css">
+
+    <div class="section-1">
+        <img class="homeimage" src="https://www.nitj.ac.in//images/cce/nitj3.jpg" alt="nitjcampusimage">
```

Comment:
Shouldn't this background image be in _header.ejs or am I missing something?

Reply:
sir the background image is meant only for the home page, that's why i have put that under the section1 of home page. According to the reference we have used i have done this. In the reference the image is only in the home page as it should be, because it is meant only for the home page. All the other pages have the navbar with background color. Only the home page is with the image behind the navbar. For home page i have specifically made the navbar transparent and achieved this whole thing using z-index property.